pkg/resource/cluster/hook.go

```diff
@@ -197,6 +197,12 @@ func (rm *resourceManager) customUpdate(
 			if !ok || awserr.Message() != LoggingNoChangesError {
 				return nil, err
 			}
+
+			// Check to see if we've raced an async update call and need to
+			// requeue
+			if ok && awserr.Code() == "ResourceInUseException" {
```

Comment:
consider making this a string const instead of hard-coding the string here (and elsewhere in this file)